fastlane/Fastfile

```diff
@@ -2,24 +2,24 @@ default_platform(:ios)
 
 platform :ios do
 
-  lane :test_iOS14_4 do
+  lane :test_iOS16 do
     run_tests(
       scheme: "Ably-iOS-Tests",
       derived_data_path: "derived_data",
-      devices: ["iPhone 12 (14.4)"],
+      devices: ["iPhone 14 (16.0)"],
```

Comment:
How come we're using iOS 16.0 and tvOS 16.1? Can't we use 16.1 for both?

Reply:
I just took value script was looking for. Will fix.

Reply:
I just took value from "Using the first of multiple matching destinations:" warning. Will fix.

Reply:
I've tried - didn't work https://github.com/ably/ably-cocoa/pull/1543/commits/83d2f98565a6caf17e9f23000ae77702715cc5ed

Reply:
What do you mean that it didn't work? I'm not sure how to use the linked commit to help me understand.

Reply:
Oh, that's weird. [Looks like](https://github.com/actions/runner-images/blob/main/images/macos/macos-12-Readme.md) Xcode 14.1 has iOS 16.1 simulators, but the default is Xcode 14.0.1. I guess since we're going with the "don't configure anything" approach, then we should leave it using Xcode 14.0.1 / iOS 16.0.